JitsiConference.js

```diff
@@ -114,17 +113,24 @@ JitsiConference.prototype.leave = function () {
     return Promise.all(
         conference.getLocalTracks().map(function (track) {
             return conference.removeTrack(track);
-        })
-    ).then(this._leaveRoomAndRemoveParticipants.bind(this))
-    .catch(function (error) {
-        logger.error(error);
-        GlobalOnErrorHandler.callUnhandledRejectionHandler(
-            {promise: this, reason: error});
-        // We are proceeding with leaving the conference because room.leave may
-        // succeed.
-        this._leaveRoomAndRemoveParticipants();
-        return Promise.resolve();
-    }.bind(this));
+        }))
+        .then(this._leaveRoomAndRemoveParticipants.bind(this))
+        .catch(function (error) {
+            logger.error(error);
+            GlobalOnErrorHandler.callUnhandledRejectionHandler(
+                {promise: this, reason: error});
+            // We are proceeding with leaving the conference because room.leave
+            // may succeed.
+            this._leaveRoomAndRemoveParticipants();
+            return Promise.resolve();
+        }.bind(this))
+        .then(function() {
```

Comment:
I thought I had some reasons for that, but now I don't see them. Code from this then() can be moved inside _leaveRoomAndRemoveParticipants() method. Also this PR should be fixed after "reload" feature was merged, so I'll take care of it a bit later.